### PR TITLE
Support GPT-4o VLM

### PR DESCRIPTION
**Pull request by kimx3966**
What does this PR do? 
---
This PR not only enables the use of the Gemini model but also allows for the use of the GPT-4 model. Specifically, you can input the desired VLM model using the --vlm argument and run the vlm.py file.

### CODE REVIEW

File: stylus/refiner/README.md [hunk 1]
```diff
@@ -21,10 +21,14 @@ python fetch_adapter_metadata.py
 
 ## 3. Added VLM descriptions to Adapter Metadata
 
-The refiner employs a VLM (Gemini-1.5) to improve the descriptions and documentations for each adapter. Our code assumes that the user has a GCP (Google Cloud) account setup.
+The refiner employs a VLM (Gemini-1.5 or GPT-4o) to improve the descriptions and documentations for each adapter. Our code assumes that the user has a GCP (Google Cloud) or OpenAI account setup.
```
Comment (michaelzhiluo) [inline stylus/refiner/README.md:24]: Thank you so much for implementing this <3 We were planning to add an experiment for ablating the refiner with different descriptions from LLMs and this is very helpful 😀

File: stylus/refiner/README.md [hunk 2]
```diff
@@ -21,10 +21,14 @@ python fetch_adapter_metadata.py
...
 The descriptions of each adapter is saved in `stylus/cache/ultra_model/[MODEL_ID]`, where `MODEL_ID` is the Civit AI model ID. Run:
 ```
-python vlm.py
+# for gemini
+python vlm.py --vlm gemini
+
+# for openai
```
Comment (michaelzhiluo) [inline stylus/refiner/README.md:30]: 💪

**Review by michaelzhiluo — APPROVED** [review]

File: stylus/refiner/README.md (hunk 1 above)
Comment (kimx3966) [reply stylus/refiner/README.md:24]: I'm glad it was helpful. I hope your experiments go well and lead to a good publication.

Comment (jwong8314) [reply stylus/refiner/README.md:24]: U the real MVP

**Pull request merged**